tests/test_loader.py

```diff
@@ -74,13 +74,13 @@ def test_load_should_create_object_from_uri_with_relative_segments():
     prefix = urlparsed.scheme + '://' + urlparsed.netloc
     expected_key_abspath = '%s%s/key.bin' % (prefix, os.path.normpath(base_uri + '/..'))
     expected_key_path = '../key.bin'
-    expected_ts1_abspath = '%s/entire1.ts' % (prefix)
+    expected_ts1_abspath = '%s%s/entire1.ts' % (prefix, base_uri)
```

Comment:
if what we want to treat is when ta malformed manifest is introducing a double `//` I think we should create a new entire test, do you agree?

Reply:
or if we want to test legit URLs with `//` I think we should create a test demonstrating that, what do you think?

Reply:
This is just a fix for new function result.

Of course new testcases may be needed, but notice that double means two or more if you add new.

Reply:
**Doesn't urljoin give you what you want?**

```python3
>>>> from urllib.parse import urljoin

>>>> baseurl="https://futzu.com//"
>>>> urljoin(baseurl,"/seg.ts")

'https://futzu.com/seg.ts'

>>>> baseurl="//"
>>>> urljoin(baseurl,"/seg.ts")

'/seg.ts'

>>>> urljoin(baseurl,"//seg.ts")

'//seg.ts'

>>>> urljoin(baseurl,"//seg.ts//")

'//seg.ts//'
```

Reply:
> Doesn't urljoin give you what you want?

It doesn't.
```python
>>> from urllib.parse import urljoin
>>> baseurl = "https://futzu.com/"
>>> urljoin(baseurl, "//seg.ts")
'https://seg.ts'
```